### PR TITLE
close #291

### LINKED ISSUE #291: changesfile "Changed-By" parameter overriden by "Maintainer"
Inside the ChangesFileBuilder:createChanges first `ChangesFile.setChanges` is called that sets the "Changed-By" to what's in the most recent changeset.

The issue is that right after that the `ChangesFile.initialize` is called which overrides it with "Maintainer" info from control file.

### PR DESCRIPTION
**Pull request by mrasko**
Fix for #291 (https://github.com/tcurdt/jdeb/issues/291) with tests.

### CODE REVIEW

File: src/test/java/org/vafer/jdeb/changes/ChangesFileBuilderTestCase.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019 Marko.koren.
+ *
```
Comment (tcurdt) [inline src/test/java/org/vafer/jdeb/changes/ChangesFileBuilderTestCase.java]: Please use "The jdeb developers." for the copyright. I will add you as contributor before the next release.

File: src/test/java/org/vafer/jdeb/changes/ChangesFileBuilderTestCase.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+import static org.junit.Assert.assertNotNull;
+import org.junit.Test;
+import org.vafer.jdeb.debian.BinaryPackageControlFile;
+import org.vafer.jdeb.debian.ChangesFile;
+
+/**
+ *
+ * @author Marko.koren
+ */
+public class ChangesFileBuilderTestCase {
```
Comment (tcurdt) [inline src/test/java/org/vafer/jdeb/changes/ChangesFileBuilderTestCase.java]: We are not using @author tags.

**Comment by tcurdt** [comment]
Thanks for the PR!

File: src/test/java/org/vafer/jdeb/changes/ChangesFileBuilderTestCase.java (hunk 1 above)
Comment (mrasko) [reply src/test/java/org/vafer/jdeb/changes/ChangesFileBuilderTestCase.java]: Forgot that netbeans adds it on new files. I'll remove it.

File: src/test/java/org/vafer/jdeb/changes/ChangesFileBuilderTestCase.java (hunk 2 above)
Comment (mrasko) [reply src/test/java/org/vafer/jdeb/changes/ChangesFileBuilderTestCase.java]: Same thing here.

**Comment by tcurdt** [comment]
Thanks!

**Pull request merged**